tests/test_sync_http_client.py

```diff
@@ -222,29 +222,11 @@ def test_bad_request_exceptions_include_expected_request_data(self):
         except BadRequestException as ex:
             assert (
                 str(ex)
-                == "(message=No message, request_id=request-123, error=example_error, error_description=Example error description)"
+                == "(message=No message, error=example_error, error_description=Example error description, foo=bar, request_id=request-123)"
             )
         except Exception as ex:
             assert ex.__class__ == BadRequestException
 
-    def test_bad_request_exceptions_exclude_expected_request_data(self):
```

Comment:
What we want is for everything in the return value to be added to exceptions, this test does the opposite.